apps/web/components/auth/TwoFactor.tsx

```diff
@@ -5,7 +5,13 @@ import { useFormContext } from "react-hook-form";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { Label, Input } from "@calcom/ui";
 
-export default function TwoFactor({ center = true }) {
+interface TwoFactorProps {
+  center?: boolean;
+  onSubmit?: (values: any, e: any) => void;
```

Comment:
can we find a type here instead of any?

Reply:
added types: [bd7f62f](https://github.com/calcom/cal.com/pull/9759/commits/bd7f62fb84178ac5fbb37ada2c3c5c08ce92d3f3?diff=split&w=0)